contrib/storage-hive/core/src/test/java/org/apache/drill/exec/hive/HiveTestUtilities.java

```diff
@@ -128,4 +128,13 @@ public static void assertNativeScanUsed(QueryBuilder queryBuilder, String table)
     assertThat(plan, containsString("HiveDrillNativeParquetScan"));
   }
 
+  /**
+   * Current Hive version doesn't support JDK 9+.
+   * Checks whether used supported for Hive Java version.
+   *
+   * @return {@code true} if used supported for Hive Java version, {@code false} otherwise
```

Comment:
Please re-phase `if used supported for Hive Java version`, similar to the comment above.